addons/website_sale/tests/test_website_editor.py

```diff
@@ -0,0 +1,105 @@
+import odoo.tests
+from odoo.tests import HttpCase, new_test_user
+import odoo.exceptions
+
+GIF = b"R0lGODdhAQABAIAAAP///////ywAAAAAAQABAAACAkQBADs="  # Tiny blank base64 image data.
+ATTACHMENT_COUNT = 5
+
+@odoo.tests.tagged('post_install', '-at_install')
```

Comment:
Might aswell import tagged directly.
```suggestion
from odoo.tests import HttpCase, new_test_user, tagged
import odoo.exceptions

GIF = b"R0lGODdhAQABAIAAAP///////ywAAAAAAQABAAACAkQBADs="  # Tiny blank base64 image data.
ATTACHMENT_COUNT = 5

@tagged('post_install', '-at_install')
```